test/application_system_test_case.rb

```diff
@@ -51,4 +51,26 @@ def select_date(selects, *values)
     selects[1].select values[1]
     selects[2].select values[2]
   end
+
+  def wait_for_ajax
+    Timeout.timeout(Capybara.default_max_wait_time) do
+      active = page.evaluate_script('jQuery.active')
+      active = page.evaluate_script('jQuery.active') until active.zero?
+    end
```

Comment:
Haha [this sounds familiar](https://github.com/panter/gsc/blob/d4419296c82bab96de71d428efe03ccf268d261a/spec/support/capybara_helper.rb#L123-L127) ;-)

You can simplify it to `loop until page.evaluate_script('jQuery.active').zero?`